cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -5020,6 +5020,26 @@
     </description>
   </property>
 
+  <property>
+    <name>security.store.system.properties.gcp-cloudkms.key.rotation.enabled</name>
+    <value>true</value>
+    <description>
+      Enable key rotation for KMS keys used to encrypt and decrypt secure keys.
+      When true, it will enable rotation for newly created keys as well as
+      existing keys (after upgrade).
+    </description>
+  </property>
+
+  <property>
+    <name>security.store.system.properties.gcp-cloudkms.key.rotation.period.in.days</name>
```

Comment:
nit: `security.store.system.properties.gcp-cloudkms.key.rotation.period.days` for consistency in property naming.